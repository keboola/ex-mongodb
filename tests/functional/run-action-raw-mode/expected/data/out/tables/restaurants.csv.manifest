{"incremental":true,"columns":["id","data"],"primary_key":["id"],"column_metadata":{"id":[{"key":"KBC.datatype.nullable","value":true}],"data":[{"key":"KBC.datatype.nullable","value":true}]}}