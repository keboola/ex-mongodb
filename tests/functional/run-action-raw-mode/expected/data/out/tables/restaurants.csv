"id","data"
"5716054bee6e764c94fa841e","{""_id"":{""$oid"":""5716054bee6e764c94fa841e""},""address"":{""building"":""1484"",""coord"":[-73.8806669,40.8283447],""street"":""Westchester Avenue"",""zipcode"":""10472""},""borough"":""Bronx"",""cuisine"":""Bakery"",""name"":""Nacional Bakery #1""}"
"5716054bee6e764c94fa8c13","{""_id"":{""$oid"":""5716054bee6e764c94fa8c13""},""address"":{""building"":""104512"",""coord"":[-73.88541699999999,40.82766],""street"":""Westchester Avenue"",""zipcode"":""10459""},""borough"":""Bronx"",""cuisine"":""Bakery"",""name"":""La Nueva Giralda Bakery""}"
"5716054bee6e764c94fa8ff6","{""_id"":{""$oid"":""5716054bee6e764c94fa8ff6""},""address"":{""building"":""1522-4"",""coord"":[-73.8789604,40.8286012],""street"":""Westchester Avenue"",""zipcode"":""10472""},""borough"":""Bronx"",""cuisine"":""Bakery"",""name"":""National Bakery""}"
